src/transform.js

```diff
@@ -699,13 +708,29 @@ const transform = {
   },
   ImportDeclaration: {
     exit(path) {
-      // TODO(#223): Handle "typeof" imports
       stripSuffixFromImportSource(path);
+      if (
+        path.node.importKind === "typeof" &&
+        t.isImportDefaultSpecifier(path.node.specifiers[0])
+      ) {
+        path.replaceWith(
+          t.tsTypeAliasDeclaration(
+            path.node.specifiers[0].local,
+            undefined,
+            t.tsTypeQuery(
+              t.tsImportType(path.node.source, t.identifier("default"))
+            )
+          )
+        );
+      }
     },
   },
   ImportSpecifier: {
     exit(path) {
-      // TODO(#223): Handle "typeof" imports
+      // TODO(#223): Handle "typeof" imports.
```

Comment:
I don't really want to spend the effort to handle `import {typeof foo, typeof bar} from "./foobar";` of `import typeof {foo, bar} from "./foobar";` because it'll be a lot of work and it seems like it shouldn't come up a lot in practice.  If anyone does use this in their code let me know.